newdle/client/src/components/Footer.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {Grid} from 'semantic-ui-react';
+import {Trans} from '@lingui/macro';
+import LanguageSelector from './common/LanguageSelector';
+import styles from './Footer.module.scss';
+
+export default function Footer() {
+  const FOOTER_LINKS = {
+    'Help': '#',
+    'Contact': '#',
+    'Terms and conditions': '#',
+    'Another link': '#',
+  };
+
+  return (
+    <footer className={styles.footer}>
+      <Grid columns={3}>
+        <Grid.Row verticalAlign="middle">
+          <Grid.Column computer={4} tablet={4} only="computer tablet">
+            <ProjectInfo />
+          </Grid.Column>
+          <Grid.Column mobile={16} tablet={8} computer={8}>
+            <p className={styles.links}>
+              {Object.keys(FOOTER_LINKS).map((key, pos, arr) => (
+                <React.Fragment key={key}>
+                  <a
+                    href={FOOTER_LINKS[key]}
+                    className={styles.link}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    {key}
+                  </a>
+                  {pos < arr.length - 1 ? <span className={styles.separator}>{' | '}</span> : null}
+                </React.Fragment>
+              ))}
+            </p>
+          </Grid.Column>
+          <Grid.Column mobile={8} only="mobile">
+            <ProjectInfo />
+          </Grid.Column>
+          <Grid.Column mobile={8} tablet={4} computer={4} textAlign="right">
+            <LanguageSelector className={styles.language} />
+          </Grid.Column>
+        </Grid.Row>
+      </Grid>
+    </footer>
+  );
+}
+
+function ProjectInfo() {
+  return (
+    <div className={styles.info}>
+      <p className={styles.newdle}>
+        <Trans>
+          Powered by{' '}
+          <a href="https://github.com/indico/newdle" target="_blank" rel="noopener noreferrer">
+            newdle
```

Comment:
Is writing `Powered by newdle` useful when we have the `newdle` name and logo in the title already? I feel like it does not really say much unless the header would be configurable (which isn't needed).
Maybe something like `newdle is Open Source` instead? What do others think about this?

Reply:
👀  @VirtualEvan

Reply:
My opinion is that "Powered by newdle" looks prettier :/
These are some of the ideas I tested

![image](https://user-images.githubusercontent.com/12986551/97492850-6a8d4800-1964-11eb-856e-82ee84308bc0.png)
![image](https://user-images.githubusercontent.com/12986551/97493600-7fb6a680-1965-11eb-9c22-fd08e03f3a8b.png)
![image](https://user-images.githubusercontent.com/12986551/97493651-93620d00-1965-11eb-8aa9-c855915b61d0.png)

Reply:
I like the "newdle is Open Source" one!

Reply:
after checking in the team, everyone agrees that the "newdle is open source" one is the preferred message - so let's go for that one :)

PS: I think the `github.svg` icon can be removed sicne we no longer use it (unless there's a nice way of keeping it in the footer)

Reply:
I added the change and rebased. Let me know if some other change is needed :)

btw: @ThiefMaster I just noticed your message is in all lowercase. Is it intended or should it be "newdle is Open Source" ?

Reply:
`newdle is Open Source` is fine. I was just too lazy to use the shift key :D